@types/route.d.ts

```diff
@@ -8,5 +8,7 @@ type AppRoute =
   | '/auth/login'
   | '/dashboard'
   | '/dashboard/families'
+  | '/dashboard/families/:id'
   | '/dashboard/families/family-detail'
+  | '/dashboard/family-detail'
```

Comment:
Why these two? `/families/:id` is enough

Reply:
they were they from the start so I leaved them there for now
but yeah you're right it shouldn't break anything